app/controllers/users_controller.rb

```diff
@@ -10,7 +10,7 @@ def create
       }
       render json: { 'logged_in': true }
     else
-      render json: { errors: user.errors.full_messages }
+      render json: { errors: user.errors.full_messages }, status: :conflict
```

Comment:
Do you want a 409 http status for email conflicts and a 400 http status for all other errors?